tools/fast_puller_.py

```diff
@@ -57,8 +66,15 @@ def main():
   if not args.name or not args.directory:
     raise Exception('--name and --directory are required arguments.')
 
+  transport_callable = httplib2.Http
+
+  if args.cacerts:
+    logging.info('Adding CA certificates of %s', args.cacerts)
+    transport_callable = _apply_ca_certs(
+      transport_callable, args.cacerts)
```

Comment:
I'd make this explicitly httplib2.Http, since none of our custom transports take that arg